runtime/builtin_types.go

```diff
@@ -193,6 +193,26 @@ func builtinAbs(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	return Abs(f, args[0])
 }
 
+func builtinMapFn(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
```

Comment:
`map()` can take multiple sequences and then supply them all as args to the function:

```
map(...)
    map(function, sequence[, sequence, ...]) -> list

    Return a list of the results of applying the function to the items of
    the argument sequence(s).  If more than one sequence is given, the
    function is called with an argument list consisting of the corresponding
    item of each sequence, substituting None for missing values when not all
    sequences have the same length.  If the function is None, return a list of
    the items of the sequence (or a list of tuples if more than one sequence).
```